GitHub-trainings/github-pr-training/latex-slides/GitHub-PR-training.tex

```diff
@@ -0,0 +1,584 @@
+			\item Most time will be spent in this stage
+			\item Reviewing the code should be seen as important as writing it
+			\item This stage will be repeated until the team agrees on the quality of the code
+		\end{itemize}
+		
+		\column{.65\textwidth} % Right column and width
+		\vspace{-.75cm}
+		\begin{figure}
+			\centering
+			\includegraphics[width=\textwidth]{./img/branch-pr-merge-cycle-S2.png}
+		\end{figure}
+		
+	\end{columns}
+\end{frame}
+
+\begin{frame}
+	\frametitle{Make commits}
+	\begin{columns}[c]
+		
+		\column{.35\textwidth} % Left column and width
+		\begin{itemize}
+			\setlength\itemsep{1em}
+			\item The author commits edits as usual
+			\item The draft PR will be updated as more commits are pushed to the branch		
+			\item Keep pushing commits until the first version implementation of the task is completed
+		\end{itemize}
+		
+		\column{.65\textwidth} % Right column and width
+		\vspace{-.75cm}
+		\begin{figure}
+			\centering
+			\includegraphics[width=\textwidth]{./img/branch-pr-merge-cycle-S2-1.png}
+		\end{figure}
+		
+	\end{columns}
+\end{frame}
+
+\begin{frame}
+	\frametitle{Update the PR}
+	\begin{columns}[c]
+		
+		\column{.35\textwidth} % Left column and width
+		\begin{itemize}
+			\setlength\itemsep{1em}
+			\item Remove the draft status from the PR
+			\item Document what has been implemented in the PR
+			\item Assign one or several people to review your PR
+			\item Notify the assignee(s) using email or slack as GitHub notifications may be muted
+		\end{itemize}
+		
+		\column{.65\textwidth} % Right column and width
+		\vspace{-.75cm}
+		\begin{figure}
+			\centering
+			\includegraphics[width=\textwidth]{./img/branch-pr-merge-cycle-S2-2.png}
+		\end{figure}
+		
+	\end{columns}
+\end{frame}
+
+\begin{frame}
+	\frametitle{Update the PR}
+	\begin{columns}[c]
+		
+		\column{.35\textwidth} % Left column and width
+		\begin{itemize}
+			\setlength\itemsep{1em}
+			\item This step is incredible important for high-quality code
+			\item Use the PR to review code edits in the \textit{file changes} tab
+			\item The reviewer adds code suggestions and comments
+		\end{itemize}
+		
+		\column{.65\textwidth} % Right column and width
+		\vspace{-.75cm}
+		\begin{figure}
+			\centering
+			\includegraphics[width=\textwidth]{./img/branch-pr-merge-cycle-S2-3.png}
+		\end{figure}
+		
+	\end{columns}
+\end{frame}
+
+
+\begin{frame}
+	\frametitle{Update the PR}
+	\begin{columns}[c]
+		
+		\column{.35\textwidth} % Left column and width
+		\begin{itemize}
+			\setlength\itemsep{1em}
+			\item The author accepts or discuss suggestions
+			\item The author replies to comments
+			\item Then the author and the reviewers (and anyone else in the project team) comes together and decides if the task is complete or not
+		\end{itemize}
+		
+		\column{.65\textwidth} % Right column and width
+		\vspace{-.75cm}
+		\begin{figure}
+			\centering
+			\includegraphics[width=\textwidth]{./img/branch-pr-merge-cycle-S2-4.png}
+		\end{figure}
```

Comment:
Show how to 
- accept suggestions
- add suggestions to batch
- reply to comments
- mark comments as resolved

Reply:
done in c75524d6497ddfc67a62cdd95ba4047c9fdc17ec - although I will only mention batch. It is not really MVO and there are so many details in this training already. I might say it or they will find it by themselves